documentation/source/building.rst

```diff
@@ -107,6 +107,10 @@ Environment Variables
     If defined, log lines displayed in terminal will be shorter. It will print only
     time, message type (``INFO``, ``WARNING``, ``ERROR``) and log message.
 
+.. envvar:: COCOTB_PDB_ON_EXCEPTION
+
+   If defined, when a python exception is hit, cocotb will drop into the python debugger.
```

Comment:
```suggestion
   If defined, cocotb will drop into the python debugger if a test fails with an exception.
```
As written it suggest that all exceptions will cause pdb, but its really just the ones that result in test failures